docs/Data_correction_documentary_cleared.ipynb

```diff
@@ -0,0 +1,299 @@
+    "|  $l^{cell}$   |  length of $^3$He-polarizer/analyzer   |  scalar, 1-dim   | from metadata   |\n",
+    "|  $T^{g, cell}(\\lambda)$   |  transmission of empty glass of polarizer/analyzer   |  scalar, 1-dim   | from metadata  |\n",
+    "|  $O^{cell}(\\lambda)=O^{0, cell}\\cdot\\lambda$   |  wavelength-dependent opacity of $^3$He-polarizer/analyzer  |  continuous function of $\\lambda$   | $O^{0, cell}$ fit to data |\n",
+    "|  $O^{0, cell}$   |  wavelength-independent pre-factor of opacity of $^3$He-polarizer/analyzer   |  scalar, 1-dim   | fit to data  |\n",
+    "|  $P^{^3He, cell}(t) = C^{cell}e^{-t/T^{1, cell}}$   |  time-dependent nuclear polarization of $^3$He-polarizer/analyzer  |  continuous function of $t$   | $C^{cell}, T^{1, cell}$ fit to data   |\n",
+    "|  $C^{cell}$   |  Pre-factor of nuclear polarization exponential decay of $^3$He-polarizer/analyzer  |  scalar, 1-dim   | fit to data | $P_{He0}$   |\n",
+    "|  $T^{1, cell}$   |  Decay constant of nuclear polarization exponential decay of $^3$He-polarizer/analyzer  |  scalar, 1-dim   | fit to data |\n",
+    "|  $I^{DB,unpol-in,cell}_{\\lambda, t}$   |  Intensity of direct-beam of an unpolarized-incoming beam to the polarized $^3$He-polarizer/analyzer  |  data array with columns for $I$, $\\lambda$, $t$   | neutron data  |\n",
+    "|  $I^{DB,depol,cell}_{\\lambda}$   |  Intensity of direct-beam of an unpolarized-incoming beam to the depolarized $^3$He-polarizer/analyzer |  data array with columns for $I$, $\\lambda$  | neutron data  ||\n",
+    "|  $I^{DB, no-cell}_{\\lambda}$   |  Intensity of direct-beam of an unpolarized-incoming beam to detector without passing cells or sample  |  data array with columns for $I$, $\\lambda$  | neutron data   |\n",
+    "|  $\\hat{T}^{\\pm, cell}(t,\\lambda)$   |  Transmission-function of a in-coming polarized neutron beam into $^3$He-polarizer/analyzer  |  Row 1   | by inserting  $P^{^3He, cell}(t)$,  $O^{cell}(\\lambda)$ |\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "e7400275-ef73-47a7-a72e-d2cdbfa1378d",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### Overview and general workflow\n",
+    "\n",
+    "Different to unpolarized SANS, the spin-dependent cross section $\\vec{S}$ after sample-scattering are correlated and cannot be corrected individually [Ref 1, 2, 3]. Instead, a beamline-component dependent correction Matrix $\\hat{M}$ has to be applied to all spin-dependent measurements simultaneously to retrieve $\\vec{S}$ from the measured intensities $\\vec{I}$:\n",
+    "\n",
+    "$$\n",
+    "\\begin{pmatrix} I^{++}_{t,\\lambda} \\\\ I^{+-}_{t,\\lambda}  \\\\ I^{-+}_{t,\\lambda} \\\\  I^{--}_{t,\\lambda}\\end{pmatrix} = \\hat{M}(t,\\lambda) \n",
+    "\\begin{pmatrix} S^{++}_{t,\\lambda} \\\\ S^{+-}_{t,\\lambda}  \\\\ S^{-+}_{t,\\lambda} \\\\  S^{--}_{t,\\lambda}\\end{pmatrix}\n",
+    "$$\n",
+    "\n",
```

Comment:
It has to be M = Ta*Tp if M is acting on S, (i.e., I = Ta*Tp*S), because these are the matrices that apply the inefficiencies from polarizer and analyzer on the sample scattering - and there first the polarizer acts on the sample, and on the resulting scattering then afterwards the analyzer again.


--> see papers (i) W. T. Lee et al., Polarisation Development at the European Spallation Source, and (ii) K. Krycka et al., Polarization-analyzed small-angle neutron scattering. I. Polarized data reduction using Pol-Corr (she did not resolve the matrix but if you insert the elements it needs to lead to the same result).

Reply:
> and there first the polarizer acts on the sample, and on the resulting scattering then afterwards the analyzer again.

This is what I thought. But according to the table, S is the corrected intensity and I the uncorrected. So why do the matrices act on the corrected? This is what seems backwards to me.

Reply:
Yes, I is uncorrected and S corrected. But I would more see it from another perspective: Imagine we start from a perfect sample scattering, i.e. we have I=S. But then, the scattering that we would measure gets changed by an inefficiency of the polarizer, then we measure what the polarizer changes on S, i.e., we measure I=T_pS, i.e., we after inserting a polarizer, measure a 'not correct' one - we measure  after the polarizer has acted on the real scattering cross sections.

Reply:
In principle one could define both, right? it only depends on how you set up T_p

Reply:
Gotcha. So the matrices encode what the polariser / analyser do, not how to correct for them (that would be the inverse operation).